PyZAPI/README.txt

```diff
@@ -0,0 +1,57 @@
+PyZAPI README
+==================
+
+Getting Started
+---------------
+
+Please ensure you are in a vagrant environment to deploy your project
+--Vagrant destroy - Destroys your current vagrant if you so require
+--Vagrant up - Starts your Vagrant 
+--Vagrant provision - Provisions vagrant
+
+--vagrant ssh - Logs you into your vagrant environment
+
+
+Afetr you have run vagrant provision, the virtualenv should of been created in home/vagrant/python3.6/
+
+--cd /home/vagrant/python3.6/
+
+Start your virtualenv
+--source bin/activate
+
+Setup your AWS credentials for Zappa
+--aws configure
+get key and secret from aws (create a user under the IAM and copy the keys)
+
+Enter the folder where the project has been created
+--cd /vagrant/
+
+install pyramid / zappa
+--pip3 install pyramid
+--pip3 install zappa
+
+create your project scaffold (already done for this project)
+--pcreate -s alchemy PyZAPI
+--cd PyZAPI
+
+Setup your pyramid
+--python3 setup.py develop Or pip install -e .
```

Comment:
Should we / could we be using/updating our scripts in the script/ folder? That’s what they’re intended for. Maybe we can do some further cleanup here to make it easier with a little shell scripting work.